### ISSUE TITLE
Desk stutters when moving

### ISSUE THREAD

**Issue by Daedren** [rhyst/linak-controller#48]
Hello! I'm trying out the program and I'm finding that the desk tends to move a bit then stop and resume moving, until it reaches the desired height. On the example below, the desk only at times reaches the full speed -62)

This is running on a Raspberry Pi 4 on Python 3.8, through Docker. It's an Ikea Idasen. Is there anything else I can provide to help out?

```
# idasen-controller --forward --move-to 670
Height:  970mm
Moving to height: 670
Height:  964mm Speed: -33mm/s
Height:  938mm Speed: -62mm/s
Height:  926mm Speed: -37mm/s
Height:  915mm Speed: -37mm/s
Height:  904mm Speed: -34mm/s
Height:  875mm Speed: -62mm/s
Height:  865mm Speed: -35mm/s
Height:  841mm Speed: -62mm/s
Height:  814mm Speed: -62mm/s
Height:  788mm Speed: -62mm/s
Height:  761mm Speed: -62mm/s
Height:  731mm Speed: -62mm/s
Height:  720mm Speed: -36mm/s
Height:  693mm Speed: -62mm/s
Height:  682mm Speed: -35mm/s
Height:  670mm Speed:  0mm/s
```

**Comment by lemisieur** [comment]
Just as an FYI, I'm getting the same shuttering running on a Mac Mini (Catalina)

**Comment by rhyst** [comment]
Fixed by #73
